docs/source/cuda._sanitizer.rst

```diff
@@ -0,0 +1,95 @@
+.. currentmodule:: torch.cuda._sanitizer
+
+torch.cuda._sanitizer
```

Comment:
The title could be in plain English, right? "CUDA stream sanitizer"?

Reply:
I thought the convention was to use the module name; but it seems the two other pages I saw that did this were the only ones. I will change it.